core/src/function.rs

```diff
@@ -361,24 +361,17 @@ impl Function {
         ret
     }
 
-    /// Begins disassembly at address `start`, inside region `reg`, with CPU state `init`.
-    pub fn dis<A: Architecture>(&mut self, init: A::Configuration, reg: &Region, start: u64) -> ()
+    /// Begins disassembly of `self` inside region `reg`, with CPU state `init`.
+    pub fn dis<A: Architecture>(&mut self, init: A::Configuration, reg: &Region) -> ()
     where
         A: Debug,
         A::Configuration: Debug,
     {
-        let maybe_entry = if let Some(ref v) = self.entry_point {
-            match self.cflow_graph.vertex_label(*v) {
-                Some(&ControlFlowTarget::Resolved(ref bb)) => Some(bb.area.start),
-                _ => None,
-            }
-        } else {
-            Some(start)
-        };
-        let (mut mnemonics, mut by_source, mut by_destination) = Self::index_cflow_graph(self.cflow_graph.clone(), maybe_entry);
+        let entry = Some(self.entry);
```

Comment:
@flanfly so removing this seems logical; the only way there could be an entry point is in the prior api we could have started, then stopped, then started again.  But now it's a one-shot computation.

However, for the single test failure, [the test directly inserts into the entry_point](https://github.com/das-labor/panopticon/blob/b05a772de4699f9ea4f210657346fe62d22987c2/core/src/function.rs#L1040); I believe this should be disallowed, or the test should be modified, what do you think?

Or perhaps; not.  I don't have deep intimate knowledge yet what precisely it's testing, so I don't know if i should modify it.